### PR TITLE
GH-1825: Add translations for the consignment status

### LINKED ISSUE #1825: Order delivery status - localization
Order data that come from OCC `/orders` endpoint contain consigments with `deliveryStatus` (and each individual consignment has its own `status`). The value is uppercased (i.e. `SHIPPED`). What are all possible values and are they always uppercased? 
obraz

### PR DESCRIPTION
**Pull request by mszczepaniak**
Closes #1825

### CODE REVIEW

File: projects/storefrontlib/src/translations/en/my-account.en.ts [hunk 1]
```diff
@@ -5,6 +5,10 @@ export const myAccount = {
     status: 'Status',
     shippedOn: 'Shipped on',
     inProcess: 'In process...',
+    READY_FOR_PICKUP: 'Ready for pickup',
```
Comment (Pucek9) [inline projects/storefrontlib/src/translations/en/my-account.en.ts]: @mszczepaniak @Platonn what do you think about have orderStatus object for that?

Comment (mszczepaniak) [reply projects/storefrontlib/src/translations/en/my-account.en.ts]: I am comfortable with both these solutions, so do not have any real preference here, how about you @Platonn ?

Comment (Platonn) [reply projects/storefrontlib/src/translations/en/my-account.en.ts]: I'd rather use special `context` translation parameter which we support https://www.i18next.com/translation-function/context . We should have it documented soon.
html:
```
{{ 'orderDetails.deliveryStatus' | cxTranslate : { context: consignment.status } }}
```

resources:
```
deliveryStatus_READY_FOR_PICKUP: 'Ready for pickup'
...
```

Then is no need to add method `addLocalizationPrefix` and store prefix in local variable of the component.

Comment (mszczepaniak) [reply projects/storefrontlib/src/translations/en/my-account.en.ts]: Ok, thank you for the information. Refactored

**Pull request merged**